Desktop/components/JASP/Widgets/FileMenu/PrefsAdvanced.qml

```diff
@@ -407,11 +407,24 @@ ScrollView
 				defaultValue:		Math.max(preferencesModel.maxEnginesAdmin, 4)
 				stepSize:			1
 
-				KeyNavigation.tab:	showEnginesWindow
+				KeyNavigation.tab:	engineSandbox
 				activeFocusOnTab:			true
 				text:				qsTr("Maximum number of engines: ")
 			}
 
+			CheckBox
+			{
+				id:					engineSandbox
+				visible:			Qt.platform.os === "windows" && preferencesModel.developerMode
```

Comment:
```suggestion
				visible:			Qt.platform.os === "windows" && (preferencesModel.developerMode || preferencesModel.engineSandbox)
```

Unless ofc it cant ever be enabled if developerMode is off

Reply:
Also if user check it on should be better to prompt a message box to ask user restart JASP.